src/components/Camera/style.css

```diff
@@ -8,70 +8,113 @@
   justify-content: space-between;
 }
 
+.unconstrainingContainer() {
+  position: static;
+  .uploadFallback {
+    display: none;
+  }
+}
+
 .video-overlay {
   /* Relatively positioned so absolute children are bound to this overlay */
   position: relative;
   overflow: hidden;
   margin: 0 40px 16px;
-}
 
-.overlayFullScreen {
-  @media (--small-viewport) {
-    position: static;
+  .livenessCamera & {
+    .unconstrainingContainer();
+  }
 
-    .uploadFallback {
-      display: none;
-    }
+  @media (--small-viewport) {
+    .unconstrainingContainer();
   }
 }
 
+.smallerRoundedBtn() {
+  width: 56px;
+  height: 56px;
+  background-color: white;
+  border: 3px solid black;
+  box-shadow: 0px 0px 0px 4px white;
+  border-radius: 50%;
+}
+
 .fullScreenBtn {
   @media (--small-viewport) {
-    width: 56px;
-    height: 56px;
-    background-color: white;
-    border: 3px solid black;
-    box-shadow: 0px 0px 0px 4px white;
-    border-radius: 50%;
+    .smallerRoundedBtn();
   }
 }
 
+.noPaddingCentredBtn {
+  padding: 0;
+  margin-left: auto;
+  margin-right: auto;
+}
+
 .btn:extend(.btn, .btn-centered, .btn-primary) {
   @media (--small-viewport) {
-    padding: 0;
-    position: absolute;
-    bottom: 48px;
-    margin-left: auto;
-    margin-right: auto;
-    left: 0;
-    right: 0;
+    .noPaddingCentredBtn();
   }
-  .btnText {
-    @media (--small-viewport) {
-      display: none;
-    }
+
+  .livenessCamera & {
+    .noPaddingCentredBtn();
+  }
+}
+
+.recordingIndicator {
+  font-size: 11px;
+  text-align: center;
+
+  /* Red circle */
+  &::before {
+    content: '';
+    display: inline-block;
+    width: 8px;
+    height: 8px;
+    margin-right: 5px;
+    background: #DC2A2A;
+    border-radius: 4px;
+    line-height: 120%;
   }
 }
 
+.iconBtn() {
```

Comment:
I think multiple selectors is better than mixins. This will repeat the css, won't it? We need to strive for the smallest css.

EDIT. SASS has "virtual" inheritance, but I don't think less has it.

Reply:
It will repeat the CSS, but I see no other way. 

Outer selectors are in scope inside `@media` in less, so `:extend`ing is not an option here 😢